client/src/routes/wikis/$wikiKey/new.tsx

```diff
@@ -38,12 +41,17 @@ const RouteComponent = () => {
       permissions={permissions}
     >
       <WikiContainer>
-        <ArticleEditor mode="create" />
+        <ArticleEditor mode="create" defaultValues={defaultValues} />
       </WikiContainer>
     </WikiProvider>
   );
 };
 
 export const Route = createFileRoute("/wikis/$wikiKey/new")({
   component: RouteComponent,
+  validateSearch: (search: Record<string, unknown>) => {
+    return {
+      categoryKey: search.categoryKey as string | undefined,
+    };
+  },
```

Comment:
je mettrais un petit validateur de zod tout simple ici, histoire d'être bien sûr qu'on a les bonnes valeurs


```suggestion
  validateSearch: (search) => z.object({ categoryKey: z.string().optional() }).parse(search),
```